config/urls.py

```diff
@@ -31,6 +31,7 @@
 
     url(r'^api/', include('metaci.api.urls')),
     url(r'^builds/', include('metaci.build.urls')),
+    url(r'^create-org/', include('metaci.create_org.urls')),
```

Comment:
sentimentally, I might ask that we keep new-org-plz as the URL path or offer it was an easter egg.

Reply:
I get the desire for sentimentality but value UX consistency (including url paths) more.  After all, we did drop the most sentimental thing from MetaCI altogether, it's original name :)